sdk/program/src/message/sanitized.rs

```diff
@@ -197,6 +197,14 @@ impl SanitizedMessage {
         }
     }
 
+    /// Returns the list of account keys used for account lookup tables.
+    pub fn message_address_table_lookups(&self) -> &[v0::MessageAddressTableLookup] {
+        match self {
+            Self::Legacy(_) => &[],
```

Comment:
nit: `_message` for little improved readability.

Reply:
f8621635898209df0936f6fc11344a98d8761610